examples/st-vault/README.md

```diff
@@ -1,9 +1,9 @@
 ## Configuration
 
-| Exposed Service              | Description               |
-|------------------------------|---------------------------|
-| `localhost:8080/prism-agent` | Single-tenant Cloud Agent |
-| `localhost:8200`             | Vault                     |
+| Exposed Service        | Description               |
+|------------------------|---------------------------|
+| `localhost:8080/agent` | Single-tenant Cloud Agent |
```

Comment:
I would replace `agent` path with `cloud-agent` to evade the questions from other people.

Reply:
fixed